src/LoginFormContainer.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import {
+  requestLogin,
+  changeLoginField,
+  logout ,
+} from './actions';
+
+import { get } from './utils';
+
+import LoginForm from './LoginForm';
+
+function LogoutForm({ onClick }) {
+  return (
+    <button
+    type="button"
+    onClick={onClick}
+    >Log out</button>
```

Comment:
```suggestion
    <button
      type="button"
      onClick={onClick}
    >
      Log out
    </button>
```

코드는 이렇게 정리해주는게 더 가독성이 좋을거에요.